src/main/java/seedu/address/model/person/Person.java

```diff
@@ -93,25 +111,29 @@ public boolean equals(Object other) {
         return name.equals(otherPerson.name)
                 && phone.equals(otherPerson.phone)
                 && email.equals(otherPerson.email)
+                && income.equals(otherPerson.income)
+                && family.equals(otherPerson.family)
```

Comment:
Great job on the indentation here as well

Reply:
great job!

Reply:
👍